### PR TITLE
refactor(colors): change from local colorSet to mirinae colorSet

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
SSIA

### CODE REVIEW

File: src/styles/colors.js [hunk 1]
```diff
@@ -1,227 +1,35 @@
...
 export const primary1 = palette.violet[400];
 export const primary2 = palette.violet[300];
 export const primary3 = palette.violet[200];
 export const primary4 = palette.violet[100];
 export const secondary = palette.blue[600];
+export const secondaryDark = palette.blue[800];
 export const secondary1 = palette.blue[500];
 export const secondary2 = palette.blue[100];
 export const alert = palette.red[500];
 export const safe = palette.green[600];
```
Comment (WANZARGEN) [inline src/styles/colors.js:35]: issue: Please use semanticColors to sync with mirinae color tokens.

**Review by piggggggggy — APPROVED** [review]
LGTM!
It' that so simple!!

**Review by WANZARGEN — APPROVED** [review]
I left a comment!

File: src/styles/colors.js (hunk 1 above)
Comment (yuda110) [reply src/styles/colors.js]: thanks I updated!

**Pull request merged**